### PR TITLE
Improve seeds

### PR DESCRIPTION
**Pull request by szymonbaranowski**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
-projects = ["Reminders", "Review", "Profile", "Access", "People"]
+projects_active = %w(Reminders Review Profile Access People)
```
Comment (lubieniebieski) [inline db/seeds.rb]: I would prefer `active_projects`, `archived_projects`, etc.

File: db/seeds.rb [hunk 2]
```diff
@@ -21,14 +24,25 @@
...
+# create projects
+projects_active.each do |project_name|
   Project.find_or_create_by(name: project_name,
                             email: "team-#{project_name.downcase}@domain.com",
                             channel_name: "project-#{project_name.downcase}")
 end
 
-#create reminders
+projects_archived.each do |project_name|
+  Project.find_or_create_by(name: project_name,
```
Comment (lubieniebieski) [inline db/seeds.rb]: why don't we create a nice factory instead?

File: db/seeds.rb [hunk 3]
```diff
@@ -21,14 +24,25 @@
...
+                            email: "team-#{project_name.downcase}@domain.com",
+                            channel_name: "project-#{project_name.downcase}",
+                            archived_at: "2010-01-01")
+end
+
+projects_disabled.each do |project_name|
+  Project.find_by_name(project_name).update_attributes(enabled: false)
```
Comment (lubieniebieski) [inline db/seeds.rb]: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits

File: db/seeds.rb (hunk 2 above)
Comment (szymonbaranowski) [inline db/seeds.rb]: I know in some of out projects we use factories in db/seeds but from what I've found online it's not recommended: 
https://robots.thoughtbot.com/factory_girl-for-seed-data

Should I still go for it?

**Comment by lubieniebieski** [comment]
:+1:

**Pull request merged**